cfg/cis-1.3/master.yaml

```diff
@@ -493,6 +493,21 @@ groups:
     scored: true
 
   - id: 1.1.30
```

Comment:
In the CIS Kubernetes 1.3 benchmark they were the other way around i.e. 1.1.30 was about API Server, and 1.1.31 was about --etcd-cafile, so the change shouldn't be applied in this 1.3 version of the file.  (I guess this explains why our tests are the wrong way around in 1.4, though I don't know why they got swapped over in the CIS benchmark!)